backend/server/src/test/java/ai/verta/modeldb/ProjectTest.java

```diff
@@ -1766,7 +1766,7 @@ public void k_getProjectByNameWithWorkspace() {
     try {
       var workspaceName = testUser2.getVertaInfo().getUsername();
       if (testConfig.isPermissionV2Enabled()) {
-        workspaceName = getWorkspaceNameUser2();
+        workspaceName = getWorkspaceNameUser1();
```

Comment:
What was the reason for this change? At a glance it looks like it should be test user 2, since a couple lines above it's getting user2.

Reply:
Instead of adding a second user as a collaborator, In Permission v2, we have the same workspace for both users in this test case.